lib/src/entities/user_profile.dart

```diff
@@ -27,6 +36,11 @@ class UserProfile with _$UserProfile {
     @JsonKey(name: 'userBioData') BioData? userBioData,
     @JsonKey(name: 'homeAddress') Address? homeAddress,
     @JsonKey(name: 'workAddress') Address? workAddress,
+
+    /// [assistant] refers to the name of the Be.Well user assistant,
+    /// i.e Bev or Bowi.
+    @JsonKey(name: 'assistant', fromJson: _decodeAssistant)
```

Comment:
The `fromJson` part of the annotation seems like an overkil.
I expected a simple implementation like this -> https://github.com/savannahghi/domain_objects/blob/add-assistant/lib/src/entities/supplier_profile.dart#L28

Reply:
Noted @daviddexter. I have utilized the alternative you suggested on this.